### ISSUE TITLE
No helper functionality for linking to another user

### ISSUE THREAD

**Issue by ecbrodie** [k0shk0sh/FastHub#1948]
**FastHub Version: 4.6.7**  
**Android Version: 8.0.0 (SDK: 26)**  
**Device Information:**  
- **Manufacturer:** OnePlus  
- **Brand:** OnePlus  
- **Model:** OnePlus3T  
---

 There are no helpers that appear when I am trying to link to or mention another user. When I type the `@` character followed by another letter, I would expect a popup or some autocomplete UI to appear. However, I have to manually type out the user's full username. In some cases, that can be pretty tedious.

**Comment by disacrol** [comment]
Yes, this is a must you comment on issues often. All GitHub Android clients I have tested do have this functionality.
